config/templates/wso2/input/inputs.conf.njk

```diff
@@ -0,0 +1,4 @@
+{% import "../../macros/input.njk" as input with context %}
+{{- input.renderTail('app.' + typeTag + '.access', logs_path_http_access_log) }}
+
+{{- input.renderMultilineTail('app.' + typeTag + '.generic', logs_path_app, 'multiline-core-app-regex-' + typeTag) }}
```

Comment:
Are there any multiline logs? Also, type should probably end in .carbon (not .generic)